reorder_python_imports.py

```diff
@@ -470,22 +469,31 @@ def _fix_file(
 REPLACES[(3, 11)].update((
     'typing_extensions=typing:Any',
     'typing_extensions=typing:LiteralString',
-    'typing_extensions=typing:NamedTuple',
     'typing_extensions=typing:Never',
     'typing_extensions=typing:NotRequired',
     'typing_extensions=typing:Required',
     'typing_extensions=typing:Self',
-    'typing_extensions=typing:TypedDict',
-    'typing_extensions=typing:Unpack',
     'typing_extensions=typing:assert_never',
     'typing_extensions=typing:assert_type',
     'typing_extensions=typing:clear_overloads',
-    'typing_extensions=typing:dataclass_transform',
     'typing_extensions=typing:final',
     'typing_extensions=typing:get_overloads',
     'typing_extensions=typing:overload',
     'typing_extensions=typing:reveal_type',
 ))
+REPLACES[(3, 12)].update((
+    'typing_extensions=typing:NamedTuple',
+    'typing_extensions=typing:Protocol',
+    'typing_extensions=typing:SupportsAbs',
+    'typing_extensions=typing:SupportsBytes',
+    'typing_extensions=typing:SupportsComplex',
+    'typing_extensions=typing:SupportsFloat',
+    'typing_extensions=typing:SupportsInt',
+    'typing_extensions=typing:SupportsRound',
+    'typing_extensions=typing:TypedDict',
+    'typing_extensions=typing:Unpack',
+    'typing_extensions=typing:dataclass_transform',
+))
```

Comment:
we tend to not add this section until it has been released at rc1 -- could you remove this chunk?

Reply:
Oh, sure